### ISSUE TITLE
Activiti 5.8 controller reloading broken in Grails 2.0.0 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [jeewan2015/grails-activiti-plugin#21]
```
Controller reloading in onChange event of the plugin broken. This error 
happened in development mode after edit Activiti-enabled controller.

Error on browser screen:
The requested resource (/.../....) is not available.

Error on console:
| Error 2011-11-25 10:48:10,344 ["http-bio-8080"-exec-2] ERROR 
filter.UrlMappingsFilter  - Error when matching URL mapping 
[/(*)/(*)?/(*)?]:null

Initial opinion: both error message has no direct indication for the root cause 
of the issue.



```

Original issue reported on code.google.com by `limchee...@vobject.com` on 17 Dec 2011 at 7:39

**Comment by GoogleCodeExporter** [comment]
```
Thanks to Keegan Key filed a similar issue at 
http://jira.grails.org/browse/GRAILS-8371

```

Original comment by `limchee...@vobject.com` on 17 Dec 2011 at 8:48

**Comment by GoogleCodeExporter** [comment]
```
Fixed by changing 'def observe = ["controllers"]' to 'def watchedResources = 
"file:./grails-app/controllers/**/*.groovy"'. To be released in 5.8.1.
```

Original comment by `limchee...@vobject.com` on 21 Dec 2011 at 3:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision fa776b7a7994.
```

Original comment by `limchee...@vobject.com` on 21 Dec 2011 at 5:10
